src/parcsr_ls/par_amg_setup.c

```diff
@@ -3288,11 +3291,27 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
          }
       }
       else if (j == num_levels - 1 &&
-               (grid_relax_type[3] == 8 || grid_relax_type[3] == 13 || grid_relax_type[3] == 14))
+               (grid_relax_type[3] == 8 || grid_relax_type[3] == 89 || grid_relax_type[3] == 13 || grid_relax_type[3] == 14))
```

Comment:
Shouldn't 88 be included here as well, just as 8 is?

Reply:
Maybe not. 88 is handled below separately as it requires type 6 for `hypre_ParCSRComputeL1Norms`